components/TechyApi.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
+import styled from 'styled-components/native';
+
+const TechyApi = () => {
+    const [techwords, setTechwords] = useState({})
+
+    const generateTechWords = () => {
+        fetch('https://techy-api.vercel.app/api/json')
+        .then(response => response.json())
+        .then((data) => setTechwords(data))
+    }
+
+const Container = styled.View`
+    flex: 1;
+	justify-content: center;
+	align-items: center;
+	margin: 20px;
+    width: 80%;
+    padding-left: 20px;
+    padding-right: 20px;  
+    `;
```

Comment:
Maybe it's just github that does that, but it seems that the indentation isn't right in some places, like here.

Reply:
Its Github for sure, it doesn´t look like this in VSC :)